cmd/helm/installer/install.go

```diff
@@ -117,8 +119,28 @@ func generateLabels(labels map[string]string) map[string]string {
 	return labels
 }
 
+// parseNodeSelectors takes a comma delimited list of key=values pairs and returns a map
+func parseNodeSelectors(labels string) map[string]string {
+	kv := strings.Split(labels, ",")
+	nodeSelectors := map[string]string{}
+	nodeSelectors["beta.kubernetes.io/os"] = "linux"
```

Comment:
This looks like a sensible way to do it. Have it default to linux and if the user specifies another one, it will be overwritten.